### PR TITLE
Feat/add mixpanel to model hooks

### PR DESCRIPTION
**Pull request by StellaMbogo**

### CODE REVIEW

File: dbt_gloss/check_model_name_contract.py [hunk 1]
```diff
@@ -44,17 +52,42 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
...
         print(f"Unable to load catalog file ({e})")
         return 1
 
-    return check_model_name_contract(
-        paths=args.filenames,
-        pattern=args.pattern,
-        catalog=catalog,
-    )
+    start_time = time.time()
+    status_code = check_model_name_contract(
```
Comment (dangkv) [inline dbt_gloss/check_model_name_contract.py:64]: I would fix the indentation here to:

```
status_code = check_model_name_contract(
    paths=args.filenames,
    pattern=args.pattern,
    catalog=catalog,
)
```

File: dbt_gloss/check_model_has_tests_by_type.py [hunk 2]
```diff
@@ -93,10 +100,29 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
             parser.error(f"Unable to cast {cnt} to int.")
         required_tests[test_type] = test_cnt
 
-    return check_test_cnt(
-        paths=args.filenames, manifest=manifest, required_tests=required_tests
-    )
+    end_time = time.time()
+    
+    status_code = check_test_cnt(paths=args.filenames,
```
Comment (dangkv) [inline dbt_gloss/check_model_has_tests_by_type.py]: Fix indentation to:
  ```
status_code = check_test_cnt(
    paths=args.filenames, 
    manifest=manifest, 
    required_tests=required_tests
)
```

File: dbt_gloss/check_model_parents_database.py [hunk 3]
```diff
@@ -80,13 +86,31 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         print("Please specify at least one `--blacklist` or `--whitelist` option.")
         return 1
 
-    return check_parents_database(
-        paths=args.filenames,
+    start_time = time.time()
+    status_code = check_parents_database(
```
Comment (dangkv) [inline dbt_gloss/check_model_parents_database.py:86]: Change indentation format

File: dbt_gloss/check_model_parents_schema.py [hunk 4]
```diff
@@ -80,13 +86,31 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         print("Please specify at least one `--blacklist` or `--whitelist` option.")
         return 1
 
-    return check_parents_schema(
-        paths=args.filenames,
+    start_time = time.time()
+    status_code = check_parents_schema(
```
Comment (dangkv) [inline dbt_gloss/check_model_parents_schema.py:86]: Change indentation format

File: dbt_gloss/check_model_parents_and_childs.py [hunk 5]
```diff
@@ -126,10 +131,27 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
             "cnt": args.max_child_cnt,
         },
     ]
-    return check_child_parent_cnt(
-        paths=args.filenames, manifest=manifest, required_cnt=required_cnt
-    )
 
+    start_time = time.time()
+    status_code = check_child_parent_cnt(paths=args.filenames, manifest=manifest,required_cnt=required_cnt)
```
Comment (dangkv) [inline dbt_gloss/check_model_parents_and_childs.py]: Change indentation format

**Comment by dangkv** [comment]
Great job @StellaMbogo, on the Mixpanel implementation.
- just a few indentations to re-format. Want to keep a strict coding standard since this project is open source
- ✅ checked all tracking descriptions, and they seemed to be matching with the hook function

**Comment by StellaMbogo** [comment]
> Great job @StellaMbogo, on the Mixpanel implementation.
> 
> * just a few indentations to re-format. Want to keep a strict coding standard since this project is open source
> * ✅ checked all tracking descriptions, and they seemed to be matching with the hook function

@dangkv I made the formatting updates you flagged and also went through all model hooks to standardize that formatt.

**Review by JFrackson — APPROVED** [review]
looks good to merge into mixpanel_sdk!

**Pull request merged**